Documentation/community/roadmap.rst

```diff
@@ -33,16 +33,17 @@ Major Feature Status
 +-----------------------------+------------------------------------------------------------+
 | Service Mesh                | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)    |
 +-----------------------------+------------------------------------------------------------+
-
+| Tetragon security           | Beta                                                      |
```

Comment:
this failed CI with malformed table:

```
3:50:46 PM: Building documentation (html)...
3:51:31 PM: Please fix the documentation warnings below
3:51:31 PM: /opt/build/repo/Documentation/community/roadmap.rst:17: ERROR: Malformed table.
3:51:31 PM: +-----------------------------+------------------------------------------------------------+
3:51:31 PM: | eBPF Networking             | Stable (:ref:`Roadmap Details<rm-advanced-networking>`)    |
3:51:31 PM: ++----------------------------+------------------------------------------------------------+
3:51:31 PM: || Kubernetes CNI             | Stable                                                     |
3:51:31 PM: ++----------------------------+------------------------------------------------------------+
3:51:31 PM: || Load Balancing             | Stable                                                     |
3:51:31 PM: ++----------------------------+------------------------------------------------------------+
3:51:31 PM: || Network Policy             | Stable                                                     |
3:51:31 PM: ++----------------------------+------------------------------------------------------------+
3:51:31 PM: || Kube-proxy Replacement     | Stable                                                     |
3:51:31 PM: ++----------------------------+------------------------------------------------------------+
3:51:31 PM: || Egress Gateway             | Stable                                                     |
3:51:31 PM: ++----------------------------+------------------------------------------------------------+
3:51:31 PM: | Multi-Cluster (ClusterMesh) | Stable (:ref:`Roadmap Details<rm-clustermesh>`)            |
3:51:31 PM: +-----------------------------+------------------------------------------------------------+
3:51:31 PM: | Hubble Observability        | Stable (:ref:`Roadmap Details<rm-hubble-observability>`)   |
3:51:31 PM: +-----------------------------+------------------------------------------------------------+
3:51:31 PM: | Service Mesh                | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)    |
3:51:31 PM: +-----------------------------+------------------------------------------------------------+
3:51:31 PM: | Tetragon security           | Beta                                                      |
3:51:31 PM: +-----------------------------+------------------------------------------------------------+
3:51:31 PM: /opt/build/repo/Documentation/community/roadmap.rst:38: WARNING: Blank line required after table.
3:51:31 PM: make: *** [Makefile:115: html-netlify] Error 1
3:51:31 PM: ​
3:51:31 PM: ────────────────────────────────────────────────────────────────
3:51:31 PM:   "build.command" failed                                        
3:51:31 PM: ────────────────────────────────────────────────────────────────
3:51:31 PM: ​
3:51:31 PM:   Error message
3:51:31 PM:   Command failed with exit code 2: make html-netlify (https://ntl.fyi/exit-code-2)
```

Reply:
:disappointed: I could have sworn I checked and the Documentation target was marked as "passed". But I can see clearly by clicking above that it's not passing. I'll revert it, then @xmulligan can try again. Maybe leave the PR as draft until you have confirmed that the CI is passing, then mark it as ready for review.